app/utils/toasts.js

```diff
@@ -2,6 +2,7 @@ const notifyFormSavedSuccessfully = (toaster) => {
   toaster.notify('Het formulier werd correct opgeslagen.', 'Succes', {
     type: 'success',
     icon: 'circle-check',
+    timeOut: 5000,
```

Comment:
got tired of seeing the success messages forever heaping up in the top right corner